USAGE.md

```diff
@@ -0,0 +1,107 @@
+# Amazon Alexa Smart Home skill for openHAB 2
+
+Amazon certified openHAB2 as a [smarthome skill for Alexa](https://www.amazon.com/openHAB-Foundation/dp/B01MTY7Z5L).
+This skill allows you to connect your openHAB setup through the myopenHAB.org cloud service to Amazon Alexa.
+With this skill you can control items that respond to on/off and numeric commands as well as limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
```

Comment:
Transform into a list?

Reply:
Do I have to? :)
I think this is fine as it is.

Reply:
No, just an idea. Leave as is. ;)